core/database/src/main/java/org/sopt/at/database/database/HomeBannerDataSourceImpl.kt

```diff
@@ -0,0 +1,130 @@
+package org.sopt.at.database.database
+
+import android.content.ContentValues
+import android.content.Context
+import android.database.sqlite.SQLiteDatabase
+import android.database.sqlite.SQLiteOpenHelper
+import org.sopt.at.database.source.HomeBannerDataSource
+import org.sopt.at.model.BannerInfo
+import javax.inject.Inject
+
+class HomeBannerDataSourceImpl @Inject constructor(private val context: Context) :
+    SQLiteOpenHelper(context, DATABASE_NAME, null, DATABASE_VERSION),
+    HomeBannerDataSource {
+
+    override fun insertBanner(category: String, title: String, image: String): Boolean {
+        val database = this.writableDatabase
+        val values = ContentValues()
+        values.put("category", category)
+        values.put("title", title)
+        values.put("image", image)
```

Comment:
여러 값을 넣어야하는 거라면 이렇게 코드를 사용할 수도 있을 것 같습니다!
```suggestion
        val values = ContentValues().apply {
           put("category", category)
           put("title", title)
           put("image", image)
        }
```

Reply:
감사합니다. 👍